core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1586,7 +1586,12 @@ class SparkContext(config: SparkConf) extends Logging {
     listenerBus.removeListener(listener)
   }
 
-  private[spark] def getExecutorIds(): Seq[String] = {
+  /**
+   * :: DeveloperApi ::
+   * Deregister the listener from Spark's listener bus.
+   */
+  @DeveloperApi
+  def getExecutorIds(): Seq[String] = {
```

Comment:
I'll move this back to private[spark] (also the comment I have seems wrong so revert the comment)